app/helpers/ineligible_reasons_helper.rb

```diff
@@ -0,0 +1,40 @@
+module IneligibleReasonsHelper
+  def ineligible_reasons(cfe_result)
+    return "" unless cfe_result.assessment_result == "ineligible"
+
+    reasons = ineligible_reasons_array(cfe_result)
+    return reasons.first if reasons.length == 1
+
+    ineligible_reasons = ":<ul class='govuk-list govuk-list--bullet'>"
+    reasons.each { |reason| ineligible_reasons += "<li>#{reason}</li>" }
+    ineligible_reasons += "</ul>"
+  end
```

Comment:
**not a blocker**, you could use `.html_safe` here since we know that no injection of html is being done from user input in this method. This should allow you to not need to `sanitize` in the view, as further below

Reply:
I tried using html_safe, but it didn't seem to work particularly easily with this use case - also it might be that using sanitise rather than html_safe is the preferred solution? - see https://apidock.com/rails/String/html_safe. I have left as is for now...